charts/seed-controlplane/charts/etcd/templates/etcd.yaml

```
@@ -0,0 +1,74 @@
+apiVersion: druid.gardener.cloud/v1alpha1
+kind: Etcd
+metadata:
+  name: etcd-{{ .Values.role }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    gardener.cloud/role: controlplane
+    role: {{ .Values.role }}
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.podAnnotations }}
+  annotations:
+{{ toYaml .Values.podAnnotations | indent 4 }}
+{{- end }}
+  selector:
+    matchLabels:
+      gardener.cloud/role: controlplane
+      app: etcd-statefulset
+      role: {{ .Values.role }}
+  labels:
+    gardener.cloud/role: controlplane
+    app: etcd-statefulset
+    role: {{ .Values.role }}
+    networking.gardener.cloud/to-dns: allowed
+    networking.gardener.cloud/to-public-networks: allowed
+    networking.gardener.cloud/to-private-networks: allowed
+  etcd:
+    tls:
+      serverTLSSecretRef:
+        name: {{ .Values.tlsServerSecretName }}
+        namespace: {{ .Release.Namespace }}
+      clientTLSSecretRef:
+        name: {{ .Values.tlsClientSecretName }}
+        namespace: {{ .Release.Namespace }}
+    metrics: {{ .Values.metrics }}
+    image: {{ index .Values.global.images "etcd" }}
+    defragmentationSchedule: {{ .Values.defragmentSchedule }}
+{{- if .Values.etcdResources }}
+    resources:
+{{ toYaml .Values.etcdResources | indent 6 }}
+{{- end }}
+    clientPort: {{ .Values.servicePorts.client }}
+    serverPort: {{ .Values.servicePorts.server }}
+  backup:
+    image: {{ index .Values.global.images "etcd-backup-restore" }}
+    port: 8080
+{{- if .Values.sidecarResources }}
+    resources:
+{{ toYaml .Values.sidecarResources | indent 6 }}
+{{- end }}
+    garbageCollectionPolicy: Exponential
+    quota: 8Gi
+    garbageCollectionPeriod: 43200s
+{{- if .Values.backupEnabled }}
+    fullSnapshotSchedule: {{ .Values.snapshotSchedule }}
+    deltaSnapshotPeriod: 300s
+    deltaSnapshotMemoryLimit: 1Gi
```

Change it to 100MiB Ref:  https://github.com/gardener/gardener-extensions/blob/master/pkg/webhook/controlplane/etcd.go#L67
```suggestion
    deltaSnapshotMemoryLimit: 100MiB
```

---

Done